src/neo/Network/P2P/MessageCommand.cs

```diff
@@ -14,10 +14,6 @@ public enum MessageCommand : byte
         GetAddr = 0x10,
         [ReflectionCache(typeof(AddrPayload))]
         Addr = 0x11,
-        [ReflectionCache(typeof(PingPayload))]
-        Ping = 0x18,
-        [ReflectionCache(typeof(PingPayload))]
-        Pong = 0x19,
```

Comment:
Why they are removed?

Reply:
Maybe we don't need ping/pong message to transfer height info after adding header back

Reply:
But they are useful to check whether the connection is alive.

Reply:
Have restored